### ISSUE TITLE
Add Gitter badge?

### ISSUE THREAD

**Issue by DavidLiedle** [ezsystems/requirements#19]
Gitter shows GitHub activity on this repo and allows IRC-like chat. Comments would still belong in the repo, but newcomers with questions could ask for advice or help in the dedicated, repo-specific Gitter. Questions not related to the ezsystems/requirements repository and its use would be directed to the [eZ Community Slack](https://ez-community-on-slack.herokuapp.com/).

Working Gitter badge below as an example of what could be placed in the README.md file:

[![Gitter](https://badges.gitter.im/ezsystems/requirements.svg)](https://gitter.im/ezsystems/requirements?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)

**Comment by bdunogier** [comment]
As much as I like gitter as a solution, indeed because of its great githun integration. I'm really worried about the fragmentation.

We can setup github integrations on slack, can't we ?

**Comment by DavidLiedle** [comment]
My thinking is that on the Community Slack, we'd still have to point questions on this repo's status, usage, what-have-you, to a dedicated channel. That channel can live here in Gitter, where there's zero barrier-to-entry (like going through the self-invite process) and a clear notion of this-topic-only. It's a place to allow newcomers to wander in, and then encourage them to join the Community Slack for more detailed and broader discussion.

We can ask about the GitHub integrations. I know we can add them, but not sure if there are available slots left to do so.

**Comment by andrerom** [comment]
Same input here as when proposed on other repos: we already have too many communication channels, imho not really an option unless we decide to move away from slack.

**Comment by bdunogier** [comment]
> like going through the self-invite process

This is an interesting argument, but do we need to risk fragmentation because of this Slack User Experience issue ? Is it something that could be improved somehow ?

Remember that not everybody may have a GitHub account, especially since you are aiming at less technical users in this repository, thus making the gitter SSO ease of use less relevant.

**Comment by andrerom** [comment]
> Remember that not everybody may have a GitHub account, especially since you are aiming at less technical users in this repository, thus making the gitter SSO ease of use less relevant.

true, but anyone wanting to contribute/participate will have to get one anyway.

**Comment by bdunogier** [comment]
An interesting alternative would be to stop using Slack as a tech' community tool, and move most of the code related talks to a Gitter. But it is not without consequences either...
